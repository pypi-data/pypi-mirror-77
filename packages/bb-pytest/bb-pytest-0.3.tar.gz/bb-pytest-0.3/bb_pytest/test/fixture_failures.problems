=================================== FAILURES ===================================
____________________________ UnitTest.test_default _____________________________

....

/usr/lib/python2.6/ConfigParser.py:540: NoOptionError
===================== 37 failed, 16 passed in 1.25 seconds =====================
