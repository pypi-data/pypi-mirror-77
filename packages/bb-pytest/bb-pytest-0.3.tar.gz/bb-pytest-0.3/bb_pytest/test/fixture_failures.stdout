============================= test session starts ==============================
platform linux2 -- Python 2.6.5 -- pytest-2.3.4 -- /usr/bin/python
plugins: xdist
collecting ... collected 53 items

...

=================================== FAILURES ===================================
____________________________ UnitTest.test_default _____________________________

....

/usr/lib/python2.6/ConfigParser.py:540: NoOptionError
===================== 37 failed, 16 passed in 1.25 seconds =====================
